index.html

```diff
@@ -13,14 +13,19 @@
       rel="stylesheet"
     />
     <link rel="stylesheet" href="style.css" />
-    <title>Raihan's Portfolio</title>
+    <title>Ibrahim's Portfolio</title>
   </head>
```

Comment:
-  [ ] On all your project cards, Kindly work on the spacing of elements, (the space between the images and the texts) to match the Figma design
![image](https://user-images.githubusercontent.com/98453979/199764763-6f7a3b9f-ab1f-4f0a-9192-e9611d8af6fb.png)

### Checkout Figma

![image](https://user-images.githubusercontent.com/98453979/199764966-8d47cbfd-d826-44af-aeca-92fb5d52e6a3.png)